integration_test/common/testdata/integration-metadata_onetrue_expected-metric/input.yaml

```diff
@@ -0,0 +1,31 @@
+app_url: "https://example.google.com/"
+short_name: OpsAgent
+long_name: Cloud OpsAgent
+description: |-
+  Example description
+minimum_supported_agent_version:
+  metrics: 1.0.0
+  logging: 1.0.0
+supported_app_version: ["Example0"]
+expected_metrics:
+- type: example.googleapis.com/example.example1
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    broker: .*
+    destination: .*
+  optional: true
+expected_logs:
+- log_name: examplelog
+  fields:
+  - name: jsonPayload.message
+    value_regex: '(example):'
+    type: string
+    description: null
```

Comment:
We can remove this part on expected_logs, as this test is testing the metrics side validation, thus less irrelevant contents in the test file, better the focus is on the metrics side

Reply:
Changed in: [ca458e4](https://github.com/GoogleCloudPlatform/ops-agent/pull/673/commits/ca458e452065931cb991b63a1550a0ba7cf88245)